academy/xl-cosmjs/multi-msg.md

```diff
@@ -464,7 +464,7 @@ When you look into the components of [`TxRaw`](https://github.com/confio/cosmjs-
 
 <HighlightBox type="info"?
 
-The keen developer will notice how in `AuthInfo.signerInfos: SignerInfo[]`, the type `SignerInfo` contains a `sequence: Long`. This sequence number indicates which transaction it is that each party is signing. This also means that Alice can _deny the service_ of the trade if she sends another transaction with the same sequence in the time it takes Bob to receive, sign and broadcast it. This could happen intentionally or not.
+The keen developer notices how in `AuthInfo.signerInfos: SignerInfo[]`, the type `SignerInfo` contains a `sequence: Long`. This sequence number indicates which transaction it is that each party is signing. This also means that Alice can _deny the service_ of the trade if she sends another transaction with the same sequence in the time it takes Bob to receive, sign and broadcast it. This could happen intentionally or not.
```

Comment:
I am not sure that the present tense works in this sentence. @rob-Hitchens 
I am used to seeing the "will notice" after a "keen developer"...